lib/cursor.js

```diff
@@ -165,6 +165,10 @@ class Cursor extends CoreCursor {
     return this.cmd.sort;
   }
 
+  set session(clientSession) {
```

Comment:
Could you bring in the helper from 4.0 and use that so we can keep setting this private?

Here it is for convenience: (I put it in `test/tools/utils.js`)
```javascript
function getSymbolFrom(target, symbolName, assertExists = true) {
  const symbol = Object.getOwnPropertySymbols(target).filter(
    s => s.toString() === `Symbol(${symbolName})`
  )[0];

  if (assertExists && !symbol) {
    throw new Error(`Did not find Symbol(${symbolName}) on ${target}`);
  }

  return symbol;
}
```

Reply:
Per our discussion I just refactored the same as I did on the 4.0 branch.